server/src/test/java/io/crate/execution/engine/aggregation/impl/StdDevAggregationTest.java

```diff
@@ -42,9 +43,9 @@
 
 public class StdDevAggregationTest extends AggregationTestCase {
 
-    private Object executeAggregation(DataType<?> argumentType, Object[][] data) throws Exception {
+    private Object executeAggregation(String name, DataType<?> argumentType, Object[][] data) throws Exception {
```

Comment:
You can revert this change, and the changes: https://github.com/crate/crate/pull/17543/files#diff-0d564d327608c83c6f9c99210b039261210563d80000654ce44771f88d472d73R78 as well.

Reply:
Good point - I have changed it